core/net/ipv6/sicslowpan.c

```diff
@@ -1438,21 +1349,22 @@ output(const uip_lladdr_t *localdest)
   framer_hdrlen = NETSTACK_FRAMER.length();
   if(framer_hdrlen < 0) {
     /* Framing failed, we assume the maximum header length */
-    framer_hdrlen = 21;
+    framer_hdrlen = 23;
   }
 #else /* USE_FRAMER_HDRLEN */
-  framer_hdrlen = 21;
+  framer_hdrlen = 23;
 #endif /* USE_FRAMER_HDRLEN */
-  max_payload = MAC_MAX_PAYLOAD - framer_hdrlen - NETSTACK_LLSEC.get_overhead();
```

Comment:
These changes look like they might be unwanted changes due to the contents of this file being copied over a newer version of the file. Because the new code still needs to compute the llsec overhead, right?